source/_posts/2017-06-14-Ansible-via-python.markdown

```diff
@@ -0,0 +1,188 @@
+# Ansible via Python ![alt text](https://cub.nobleprog.com/sites/hitramx/files/styles/height50_scale/public/category_image/cursos-de-ansible-en-mexico.png?itok=xPUrGNrA)
+
+**_Ansible is designed around the way people work and the way people work together_**
+
+## What is Ansible
+
+Ansible is simple IT engine for automation, it is designed for manage many
+systems, rather than just one at a time. Ansible automates cloud provisioning,
+configuration management, application deployment, intra-service
+orchestration, and many other IT operations. It is easy to deploy due to using
+no agents and no additional custom security infrastructure. We can define own
+configuration management in simple YAML language, which is named
+ansible-playbook. YAML is easier to read and write by humans than other common
+data formats line XML or JSON. Futhermore, most programming languages contain
+libraries which operate and work YAML.
+
+## Inventory
+
+Ansible works collaterally on many systems in your infrastructure, so it is
+important to specify a roster to keep `hosts`. This list is named `inventory`,
+which can be in one of many formats. For this example, the format is an
+`INI-like` and is saved in `/etc/ansible/hosts`.
+
+```INI
+mail.example.com
+
+[webservers]
+foo.example.com
+bar.example.com
+
+[dbservers]
+one.example.com
+two.example.com
+three.example.com
+```
+
+## Ansible Playbook
+
+Playbook is the way to direct systems, which is particularly powerful, compact
```

Comment:
@Macias118 not sure if `direct` is correct in this context. Did you mean to manage systems?